Dockerfile

```diff
@@ -14,3 +14,5 @@ RUN npm ci
 
 # Copy the rest of the repo's code
 COPY . .
+
+curl $SNITCH_URL
```

Comment:
Will this ever cause any issues if `$SNITCH_URL` is empty? invoking `curl` without any arguments gives an exit code of 2.

Reply:
I don't believe so, since `SNITCH_URL` is set as a bespoke secret on this repository